hazelcast/src/main/java/com/hazelcast/spi/impl/proxyservice/impl/ProxyRegistry.java

```diff
@@ -91,16 +90,8 @@ boolean contains(String name) {
      * @param result The ProxyInfo of all proxies in this registry.
      */
     public void getProxyInfos(Collection<ProxyInfo> result) {
-        for (Map.Entry<String, DistributedObjectFuture> entry : proxies.entrySet()) {
-            final DistributedObjectFuture future = entry.getValue();
-            if (!future.isSet()) {
-                continue;
-            }
-            final DistributedObject distributedObject = future.get();
-
-            if (distributedObject instanceof InitializingObject) {
-                result.add(new ProxyInfo(serviceName, entry.getKey()));
-            }
+        for (String proxyName : proxies.keySet()) {
+            result.add(new ProxyInfo(serviceName, proxyName));
```

Comment:
we don't care if the proxy was already resolved or not. name is the only relevant part.